docs/tutorials/semantic_search/conversation_search_with_byte_quantized_vector.md

```diff
@@ -0,0 +1,278 @@
+# Topic
+
+> This tutorial doesn't explain byte-quantized vectors in detail. For more information, see [Byte-quantized vectors in OpenSearch](https://opensearch.org/blog/byte-quantized-vectors-in-opensearch/).
+
+This tutorial shows how to build semantic search using the [Cohere Embed model](https://docs.cohere.com/reference/embed) and byte-quantized vectors in OpenSearch.
+
+The Cohere Embed v3 model supports several `embedding_types`. This tutorial uses the `int8` type for byte-quantized vectors. 
+
+Note: Replace the placeholders that start with `your_` with your own values.
+
+# Steps
+
+The Cohere Embed v3 model supports several input types. This tutorial uses the following input types (from the Cohere [documentation](https://docs.cohere.com/reference/embed)):
+> - `search_document`: Used for embeddings stored in a vector database for search use cases.
```

Comment:
These should both be quoted, or neither quoted.

Reply:
good catch, will fix